manifests/init.pp

```diff
@@ -118,12 +118,10 @@
     }
   }
 
-  anchor { 'java::begin:': }
-  -> package { 'java':
+  package { 'java':
     ensure          => $version,
     install_options => $package_options,
     name            => $use_java_package_name,
   }
   -> class { 'java::config': }
```

Comment:
`java::config` used to be contained by `java` with this anchor pattern.  Following this change it isn't anymore.  Therefore if anyone has any calling code that eg. did a `require java` or `service {'foo': require => 'java'}` the configuration of java is now not guaranteed to happen first.